Dockerfile

```diff
@@ -34,33 +57,29 @@ ENV DEPS \
 # https://stackoverflow.com/questions/67596193/building-a-multi-architecture-docker-image-but-dockerfile-requires-different-pa
 ENV INSTALL_DEPS \
     wget
-RUN apt-get update && \
-    apt-get install --no-install-recommends --no-install-suggests --yes \
-    ${DEPS} ${INSTALL_DEPS}
+RUN apt-get install --quiet --quiet --yes \
+    --no-install-recommends --no-install-suggests \
+    $DEPS $INSTALL_DEPS
 
 ###
-# Setup the unprivileged user and its home directory
+# Make sure pip and setuptools are the latest versions
+#
+# Note that we use pip --no-cache-dir to avoid writing to a local
+# cache.  This results in a smaller final image, at the cost of
+# slightly longer install times.
 ###
-ARG CISA_GID=421
-ARG CISA_UID=${CISA_GID}
-ENV CISA_USER="cisa"
-ENV CISA_GROUP=${CISA_USER}
-ENV CISA_HOME="/home/cisa"
+RUN pip install --no-cache-dir --upgrade pip setuptools
```

Comment:
This section should be removed because we install pinned versions:
https://github.com/cisagov/guacscanner-docker/blob/f8ee358787025e635169aab33cc9e137bbbc2b45/Dockerfile#L86-L90

Reply:
Ah right, sorry.